### PR TITLE
Change the amplitude script

### PR DESCRIPTION
**Pull request by qdequele**
I just updated the amplitude script.

### CODE REVIEW

File: .vuepress/config.js [hunk 1]
```diff
@@ -540,43 +531,10 @@ module.exports = {
...
-        analytics.load("jezyOXRIO1Azyxx0vCcH1afz4b4boYmp");
-        analytics.page();
-      }}();
-    `,
-    ],
+    ]
```
Comment (qdequele) [inline .vuepress/config.js:538]:
```suggestion
    ],
```

**Comment by dichotommy** [comment]
@gmourier I don't know the first thing about reviewing this, would you be able to have a look? 🙏🏻

**Comment by dichotommy** [comment]
Hello @mdubus , I know you have some experience with Amplitude; is there any chance you would be able to review this PR? I would be super grateful 🙏🏻 🧁

**Comment by qdequele** [comment]
I converted it to a draft PR because I have to check how Amplitude collects data and which data by default.

**Comment by mdubus** [comment]
@dichotommy yes of course ! @qdequele just request me as a reviewer once it's ready and I'll review it :)

**Comment by maryamsulemani97** [comment]
Hi @qdequele, is this PR ready for review, or will you be making any more changes? 
Thanks!

**Comment by gmourier** [comment]
Sorry, @dichotommy, I forgot to answer you; I needed more information to be helpful; I should have asked for some right away! 🙇‍♂️

@qdequele, could you please describe the underlying idea for that PR? It will be super helpful to share some context. Thanks! 🙇‍♂️

**Comment by qdequele** [comment]
Hello everyone, it was just a draft I will delete it. It was just a test.

**Pull request closed without merging**